bioconda_utils/cli.py

```diff
@@ -98,6 +103,24 @@ def build(recipe_folder,
     else:
         docker_builder = None
 
+    # handle git range
+    if git_range:
+        p = sp.run(['git', 'diff',
+                          '--relative=recipes',
+                          '--name-only'] + git_range +
+                         [os.path.join(recipe_folder, '*', 'meta.yaml'),
```

Comment:
How can this be less explicit? It is way more explicit and you are way more flexible in your folder structure. Glob'ing is everything else than explicit.